dashboard/app/models/script_level.rb

```diff
@@ -136,12 +136,8 @@ def end_of_script?
   end
 
   def long_assessment?
-    if assessment
-      if level.properties["pages"] && level.properties["pages"].length > 1
-        return true
-      end
-    end
-    false
+    return false unless assessment
```

Comment:
Good point, and a quick test implies that would be the case.  While things would likely still work, agreed that it would be best to keep this functional explicitly returning `true` and `false`.